File: win-prog/ProgWinCS/Pages and Transforms/HundredPixelsSquare/HundredPixelsSquare.cs
//--------------------------------------------------
// HundredPixelsSquare.cs © 2001 by Charles Petzold
//--------------------------------------------------
using System;
using System.Drawing;
using System.Windows.Forms;

class HundredPixelsSquare: PrintableForm
{
     public new static void Main()
     {
          Application.Run(new HundredPixelsSquare());
     }
     public HundredPixelsSquare()
     {
          Text = "Hundred Pixels Square";
     }
     protected override void DoPage(Graphics grfx, Color clr, int cx, int cy)
     {
          grfx.FillRectangle(new SolidBrush(clr), 100, 100, 100, 100);
     }
}

File: win-prog/ProgWinVB/Pages and Transforms/HundredPixelsSquare/HundredPixelsSquare.vb
'----------------------------------------------------
' HundredPixelsSquare.vb (c) 2002 by Charles Petzold
'----------------------------------------------------
Imports System
Imports System.Drawing
Imports System.Windows.Forms

Class HundredPixelsSquare
    Inherits PrintableForm

    Shared Shadows Sub Main()
        Application.Run(new HundredPixelsSquare())
    End Sub

    Sub New()
        Text = "Hundred Pixels Square"
    End Sub

    Protected Overrides Sub DoPage(ByVal grfx As Graphics, _
            ByVal clr As Color, ByVal cx As Integer, ByVal cy As Integer)
        grfx.FillRectangle(New SolidBrush(clr), 100, 100, 100, 100)
    End Sub
End Class
